packages/components/src/components/hds/accordion/item/button.hbs

```diff
@@ -10,7 +10,9 @@
   aria-controls={{@contentId}}
   aria-expanded={{if @isOpen "true" "false"}}
   aria-label={{@ariaLabel}}
+  aria-labelledby={{@ariaLabelledBy}}
   ...attributes
 >
+  <div id={{@id}} aria-hidden="true" class="sr-only">Expand</div>
```

Comment:
Would it be possible/practical to update the Button aria-label instead of adding a nested element?

If you do use a nested element like this it should be changed to a `span` tag as block-level tags shouldn't be nested inside HTML buttons as they are inline-level.